src/web/lib/components/ThemeCustomBackgroundPicker/index.js

```diff
@@ -19,6 +19,21 @@ import "./index.scss";
 import iconHAlignLeft from "./icon_align_left.svg";
 import iconVAlignCenter from "./icon_align_center.svg";
 
+
+const mapErrors = ({ tooLarge, wrongType}) => {
+  if (tooLarge && wrongType) {
+    return `
```

Comment:
This function is used both for generating the HTML and title attribute. HTML is insensitive to line breaks (treated as whitespace), but the title attribute is not. I think that with this change, you may end up with a tooltip that has a leading and trailing blank line.

To account for this, how about returning an array here, and let the callers format the message as they see fit? For example:

HTML:

```
mapErrors(errors).map(msg => `<li>${msg}</li>`);
```

Title:
```
mapErrors(errors).join("\n")
```

Reply:
yep :) i agree that's a lot better!